javascript/src/detectCiEnvironment.ts

```diff
@@ -1,3 +1,5 @@
+import { URL } from 'url'
```

Comment:
Why is this imported? Node’s URL class is deprecated. We should use the global `URL` class.

Reply:
No, Nod's `url` class is deprecated. This is the `URL` one which is recommended.
It is available globally, but typescript was not happy with this. It is possible to explicitly import it from the `url` module as described in the documentation: https://nodejs.org/dist/latest-v16.x/docs/api/url.html#new-urlinput-base